### PR TITLE
Implement lobby chat and add comments

### PR DESCRIPTION
**Pull request by HitRam**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -9,37 +10,54 @@ app.set('view engine', 'ejs');
...
+
+			//display message to all clients in lobby including sender
 			io.sockets.emit('Display Message', {msg: data, user: socket.username});
 		}
 	});
 
 	//When user disconnets remove user from users
 	socket.on('disconnect', function() {
+
+		//display to all users (except sender) in room that user left
```
Comment (abhigyank) [inline app.js:60]: Should be leaver right, rather then sender?

File: views/index.ejs [hunk 2]
```diff
@@ -29,77 +29,19 @@
...
-                    <span class="name">Hitesh Ramachandani</span>
-                    <span class="time">1:44 PM</span>
-                    <span class="preview">Lorem ipsum dolor sit amet</span>
-                </li>
-                <li class="person" data-chat="person3">
-
-                    <span class="name">Lakshay arora</span>
-                    <span class="time">2:09 PM</span>
-                    <span class="preview">Lorem ipsum dolor sit amet</span>
+                    <span class="preview">What should we display here....</span>
```
Comment (abhigyank) [inline views/index.ejs:34]: Maybe description of Lobby? 
"Global common room"?

Comment (HitRam) [reply views/index.ejs:34]: Yeah cool.
But in next commit ok?

File: app.js (hunk 1 above)
Comment (HitRam) [reply app.js:60]: I guess won't matter but if you say so
(because sender in my sense is the client who emitted this)

**Pull request merged**